geoengine/workflow.py

```diff
@@ -378,10 +379,19 @@ def get_xarray(
             timeout,
             force_no_data_value
         ) as memfile, memfile.open() as dataset:
-            data_array = xr.open_rasterio(dataset)
+            data_array = rioxarray.open_rasterio(dataset)
 
-            # TODO: add time information to dataset
+            # helping mypy with inference
+            assert isinstance(data_array, DataArray)
 
+            rio: DataArray = data_array.rio
+            rio.update_attrs({
+                'crs': rio.crs,
+                'res': rio.resolution(),
+                'transform': rio.transform(),
+            }, inplace=True)
+
+            # TODO: add time information to dataset
```

Comment:
Do you know if we can set something, here?

Reply:
Not sure which line you are referring to? Do you mean line 391? In that case we can call [rio.write_transform()](https://corteva.github.io/rioxarray/html/rioxarray.html#rioxarray.rioxarray.XRasterBase.write_transform).

Reply:
I'm referring to the TODO, to add time information.

Reply:
lets do this as a different task